pkg/registry/core/pod/strategy.go

```diff
@@ -535,3 +536,12 @@ func PortForwardLocation(
 	}
 	return loc, nodeInfo.Transport, nil
 }
+
+// ExtractOwnerUID concats all ownerReferences uid to a string
+func ExtractOwnerUID(m []metav1.OwnerReference) (fmtStr string) {
+	for _, ownerRef := range m {
+		fmtStr += fmt.Sprintf("%v\n", string(ownerRef.UID))
```

Comment:
I don't see how this is going to be useful since the client has to list all the UIDs in the right order. Our filtering stack is not built for selecting an object if anything in a list matches, which I think is what you actually want.